jenkins/bootstrap.py

```diff
@@ -267,7 +315,7 @@ def finish(gsutil, paths, success, artifacts, build, version, repo):
         'passed': bool(success),
         'metadata': metadata(repo, artifacts),
     }
-    gsutil.upload_json(paths.finished, data)
+    gsutil.upload_json(paths.finished, data)  # Raise if this fails
```

Comment:
if this fails, we'll still abort and fail to upload the build log? is that intentional?